### ISSUE TITLE
Support for Windows 10 Mobile

### ISSUE THREAD

**Issue by assetsense** [brody2consult/cordova-sqlite-storage#732]
Does this plugin support 'Windows 10 Mobile'? I believe Microsoft replaced Windows Phone OS with Windows 10 Mobile.

**Comment by brody2consult** [comment]
Yes, just build for ARM (32-bit) to support Windows 10 Mobile ref: <https://en.wikipedia.org/wiki/Windows_10_Mobile>

My understanding had been that Windows 10 UWP would support Windows 10 desktop, tablet, and mobile. Please correct me in case I am mistaken here.

Please report if you encounter any issues with this.

**Comment by dan-alarcon** [comment]
Hello Chris, thanks for backing up to the windows platform. Your plugin is the only one that works. I wish you a great year! Cheers
